tests/e2e/tests/content-manager/relations-on-the-fly/edit-relation.spec.ts

```diff
@@ -8,7 +8,7 @@ const AUTHOR_EDIT_URL =
 
 test.describe('Relations on the fly - Edit a Relation', () => {
   test.beforeEach(async ({ page }) => {
-    await resetDatabaseAndImportDataFromPath('with-admin.tar');
+    await resetDatabaseAndImportDataFromPath('rotf.tar');
```

Comment:
Is this intentional or was this for testing? Idk if we want to introduce a new dataset

Reply:
I changed the with-admin dataset, I added a new Author user with the permission I needed to check the creatable disabled button, let me know if it is ok now, thanks